node/tests/k8st/infra/calico-kdd.yaml

```diff
@@ -160,14 +4102,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: calico-node
 rules:
-  # The tests require that node has access to secrets for BGP peering.
```

Comment:
Need to provide these permissions some other way - e.g. with a second ClusterRole / Binding.